### PR TITLE
Fix KeyError: 'designation' error oversight

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd

### CODE REVIEW

**Comment by caesar2164** [comment]
also, @dcadams

**Comment by stvstnfrd** [comment]
What's not clear to me is why we would need to use this syntax here. Other optional arguments,  like `grade_value` and `template_file` [1] aren't written this way.

[1] https://github.com/Stanford-Online/edx-platform/blob/master/lms/djangoapps/certificates/management/commands/regenerate_user.py#L131,L132

Further, if the issue is a `KeyError` with this key not existing in the dict, I'm not sure how this line wouldn't break either.

File: lms/djangoapps/certificates/management/commands/regenerate_user.py [hunk 1]
```diff
@@ -108,7 +108,9 @@ def handle(self, *args, **options):
 
         course = modulestore().get_course(course_id, depth=2)
 
-        designation = options['designation']
+        designation = 'None'
+        if options['designation']:
+            designation = options['designation']
```
Comment (stvstnfrd) [inline lms/djangoapps/certificates/management/commands/regenerate_user.py]: I think these 3 lines can be simplified to:

```python
designation = options['designation'] or 'None'
```

**Comment by stvstnfrd** [comment]
+1

**Pull request merged**